### PR TITLE
[MRG, Minor] Disallow invert_y input to epochs.plot_image

### LINKED ISSUE #5240: invert_y is broken in epochs.plot_image
:|

### PR DESCRIPTION
**Pull request by jona-sassenhagen**
closes #5240

Eventually, `epochs.plot_image` should be able to handle y axis inversion for the ERP, but I don't have the capacity to handle it right now.

### CODE REVIEW

File: mne/viz/epochs.py [hunk 1]
```diff
@@ -173,6 +174,10 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
     else:
         picks = np.atleast_1d(picks)
 
+    if "invert_y" in ts_args:
+        raise NotImplementedError("'invert_y' found in 'ts_args'. "
+                                  "This currently")
```
Comment (larsoner) [inline mne/viz/epochs.py]: Is this some sort of existential statement?

File: mne/viz/tests/test_epochs.py [hunk 2]
```diff
@@ -171,6 +171,7 @@ def test_plot_epochs_image():
                       ts_args=ts_args)
         warnings.simplefilter('always')
     assert_equal(len(w), 1)
+    epochs.plot_image(ts_args=dict(invert_y=True))
```
Comment (larsoner) [inline mne/viz/tests/test_epochs.py]: `with pytest.raises(NotImplementedError, match='currently')`

Comment (jona-sassenhagen) [reply mne/viz/tests/test_epochs.py]: This is the worst PR I've ever done :D

File: mne/viz/epochs.py (hunk 1 above)
Comment (jona-sassenhagen) [reply mne/viz/epochs.py]: Damnit! The syntax is still broken!

**Pull request merged**